LICENSE

```diff
@@ -22,3 +50,28 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
 For more information, please refer to <http://unlicense.org/>
+
+MIT license
+-----------
+The MIT License (MIT)
+
+Copyright (c) 2013-2017 Luke Sampson (https://github.com/lukesampson)
+Copyright (c) 2013-present Scoop contributors (https://github.com/ScoopInstaller/Scoop/graphs/contributors)
```

Comment:
This means that Scoop has been started with an MIT license since 2013, but not actually.

Reply:
Then I'll modify the LICENSE to change `2013-present` to `2022-present`, and zalgoize the copyright sign for Luke Sampson since while the repo was called `lukesampson/scoop`, the Unlicense was in use.

Reply:
Not a lawyer, so this is not legal advice, but my understanding of copyright law, at least as it's applied in the US:

A copyright begins the [moment the work is created](https://www.wpi.edu/about/policies/copyright-compliance/copyright-law#:~:text=The%20Copyright%20Act%20provides%20that,protected%20under%20U.S.%20copyright%20law.). Also, the copyright notice is [not required](https://www.newmediarights.org/business_models/artist/do_i_need_put_copyright_notice_my_work) for a copyright claim to be valid.

So adding the MIT license does not start the copyright clock, as the entire body of work was already copyrighted the moment each body of the work is created, and is held by each individual contributor (at least in the US).

In effect, whether a user accepts to be bound by the MIT license or not, the body of work has always been copyrighted from the date of each contribution (at least in the US).

My guess is most IP lawyers would say that the Unlicense's claiming the work is in the public domain is not legally possible, as by simply creating a work, it's copyrighted, and the only way to remove that copyright is to transfer it to another, or wait until it [enters to public domain](https://en.m.wikipedia.org/wiki/Public_domain_in_the_United_States) 95 years after its creation (at least in the US). 

Given that, they would probably argue that regardless of any user's claim of accepting only the Unlicense, only the MIT license applies to them. If we didn't add the MIT license, they might argue the work isn't really open source, but closed source, since the Unlicense does really license anything.

Again, not a lawyer, so not legal advice, but I did sue a city, not once, but twice, winning both, [getting an apology and US$6,000](https://www.lincolncity.org/Home/Components/News/News/14/16), which I donated to a local homeless shelter.

Reply:
I'm also not a lawyer but that matches my understanding @rasa. I appreciate all your efforts on this, thank you.